phraser/cc/pyext/phraserext.cpp

```diff
@@ -189,6 +189,8 @@ PyObject* MakeDict(const vector<PyObject*>& keys,
         if (PyDict_SetItem(d, k, v)) {
             return NULL;
         }
+        Py_DECREF(k);
+        Py_DECREF(v);
```

Comment:
Alternatively these DECREF calls can be made in the function that calls MakeDict, but that is more tedious work.